e2e/cypress/integration/integrations/builtin_commands/common_commands_spec.js

```diff
@@ -72,6 +73,30 @@ describe('I18456 Built-in slash commands: common', () => {
             cy.get(`#postMessageText_${postId}`).should('have.text', 'test ¯\\_(ツ)_/¯');
         });
     });
+
+    it('MM-T2345 /me on RHS', () => {
+        loginAndVisitDefaultChannel(user1, testChannelUrl);
+        cy.postMessage(MESSAGES.MEDIUM);
+
+        // # Open RHS (reply thread)
+        cy.clickPostCommentIcon();
+
+        // # type /me test
+        cy.get('#reply_textbox').type('/me test');
+        cy.get('#addCommentButton').click();
+        cy.uiWaitUntilMessagePostedIncludes('test');
+
+        cy.getLastPostId().then((postId) => {
```

Comment:
I think we need to verify the current user from UI and not from API. Since postId is the same for RHS and main channel, we can infer that the user for RHS post is the same user for main channel. This block can be re-written to,

```
        cy.getLastPostId().then((postId) => {
            // * Verify RHS message is from current user and properly formatted with lower opacity
            cy.get(`#rhsPost_${postId}`).should('have.class', 'current--user').within(() => {
                cy.get('button').should('have.text', user1.username);
                cy.get('p').should('have.text', 'test').and('have.css', 'color', 'rgba(61, 60, 64, 0.6)');
            });

            // * Verify main channel message is properly formatted with lower opacity
            cy.get(`#postMessageText_${postId}`).contains('p', 'test').should('be.visible').and('have.css', 'color', 'rgba(61, 60, 64, 0.6)');
        });
```